diff --git a/catalyst/base/stagebase.py b/catalyst/base/stagebase.py
index 80cbbe74..02300c84 100644
--- a/catalyst/base/stagebase.py
+++ b/catalyst/base/stagebase.py
@@ -88,7 +88,6 @@ class StageBase(TargetBase, ClearBase, GenBase):
             self.unpack,
             self.config_profile_link,
             self.setup_confdir,
-            self.process_repos,
         ]
         self.build_sequence = [
             self.bind,
@@ -942,17 +941,22 @@ class StageBase(TargetBase, ClearBase, GenBase):
                 'Resume point detected, skipping setup_confdir operation...')
             return
 
+        dest = normpath(
+            self.settings['chroot_path'] + '/' + self.settings['port_conf'])
+        if os.path.exists(dest):
+            for item in os.listdir(dest):
+                if item != 'make.profile':
+                    clear_path(os.path.join(dest, item))
+
         if "portage_confdir" in self.settings:
             log.info('Configuring %s...', self.settings['port_conf'])
-            dest = normpath(
-                self.settings['chroot_path'] + '/' + self.settings['port_conf'])
             ensure_dirs(dest)
             # The trailing slashes on both paths are important:
             # We want to make sure rsync copies the dirs into each
             # other and not as subdirs.
             cmd(['rsync', '-a', self.settings['portage_confdir'] + '/', dest + '/'],
                 env=self.env)
-            self.resume.enable("setup_confdir")
+        self.resume.enable("setup_confdir")
 
     def to_chroot(self, path):
         """ Prepend chroot path to the given path. """
@@ -1161,23 +1165,6 @@ class StageBase(TargetBase, ClearBase, GenBase):
             os.rename(hosts_file, hosts_file + '.catalyst')
             shutil.copy('/etc/hosts', hosts_file)
 
-        # write out the make.conf
-        try:
-            self.write_make_conf(setup=True)
-        except OSError as e:
-            raise CatalystError('Could not write %s: %s' % (
-                normpath(self.settings["chroot_path"] +
-                         self.settings["make_conf"]), e)) from e
-
-        # write out the binrepos.conf
-        # we do this here for later user convenience, but normally
-        # it should not affect stage builds (which only get --usepkg,
-        # but never --getbinpkg as emerge parameters).
-        try:
-            self.write_binrepos_conf()
-        except OSError as e:
-            raise CatalystError('Could not write binrepos.conf: %s' % ( e )) from e
-
         self.resume.enable("chroot_setup")
 
         # ensure that code running in the chroot can access the jobserver
@@ -1380,11 +1367,6 @@ class StageBase(TargetBase, ClearBase, GenBase):
             if name in self.settings['keep_repos']:
                 continue
 
-            # Remove repos.conf entry
-            repo_conf = self.get_repo_conf_path(name)
-            chroot_repo_conf = self.to_chroot(repo_conf)
-            chroot_repo_conf.unlink(missing_ok=True)
-
             # The repo has already been unmounted, remove the mount point
             location = self.get_repo_location(name)
             chroot_location = self.to_chroot(location)
@@ -1395,10 +1377,6 @@ class StageBase(TargetBase, ClearBase, GenBase):
             cmd(['chown', '-R', 'portage:portage',
                 self.settings['stage_path']+self.settings['repo_basedir']])
 
-        if "sticky-config" not in self.settings["options"]:
-            # re-write the make.conf to be sure it is clean
-            self.write_make_conf(setup=False)
-
         # Clean up old and obsoleted files in /etc
         if os.path.exists(self.settings["stage_path"]+"/etc"):
             cmd(['find', self.settings['stage_path'] + '/etc',
diff --git a/targets/stage1/chroot.sh b/targets/stage1/chroot.sh
index 1eb884b5..c377f756 100755
--- a/targets/stage1/chroot.sh
+++ b/targets/stage1/chroot.sh
@@ -27,10 +27,6 @@ FEATURES="${FEATURES} nodoc noman noinfo"
 sed -i -e 's:BINPKG_COMPRESS="bzip2":BINPKG_COMPRESS="zstd":' \
 	/usr/share/portage/config/make.globals
 
-# We need to ensure the base stage3 has USE="bindist"
-# if BINDIST is set to avoid issues with openssl / openssh
-[ -e ${clst_make_conf} ] && echo "USE=\"${BINDIST} ${USE}\"" >> ${clst_make_conf}
-
 # Update stage3
 if [ -n "${clst_update_seed}" ]; then
 	if [ "${clst_update_seed}" == "yes" ]; then
@@ -51,17 +47,12 @@ else
 	echo "Skipping seed stage update..."
 fi
 
-# Clear USE
-[ -e ${clst_make_conf} ] && sed -i -e "/^USE=\"${BINDIST} ${USE}\"/d" ${clst_make_conf}
-
 export ROOT="${clst_root_path}"
 mkdir -p "$ROOT"
 
 ## START BUILD
 # First, we drop in a known-good baselayout
-[ -e ${clst_make_conf} ] && echo "USE=\"${USE} build\"" >> ${clst_make_conf}
-run_merge --oneshot --nodeps sys-apps/baselayout
-sed -i "/USE=\"${USE} build\"/d" ${clst_make_conf}
+USE="${USE} build" run_merge --oneshot --nodeps sys-apps/baselayout
 
 echo "$locales" > /etc/locale.gen
 for etc in /etc "$ROOT"/etc; do
@@ -69,18 +60,6 @@ for etc in /etc "$ROOT"/etc; do
 done
 update_env_settings
 
-# Now, we install our packages
-if [ -e ${clst_make_conf} ]; then
-	echo "CATALYST_USE=\"-* build ${BINDIST} ${clst_CATALYST_USE}\"" >> ${clst_make_conf}
-	echo "USE=\"\${CATALYST_USE} ${USE} ${BOOTSTRAP_USE} ${clst_HOSTUSE}\"" >> ${clst_make_conf}
-
-	for useexpand in ${clst_HOSTUSEEXPAND}; do
-		x="clst_${useexpand}"
-		echo "${useexpand}=\"${!x}\"" \
-		>> ${clst_make_conf}
-	done
-fi
-
 run_merge --implicit-system-deps=n --oneshot "${buildpkgs[@]}"
 
 # TODO: Drop this when locale-gen in stable glibc supports ROOT.
@@ -95,14 +74,3 @@ if [ -x "$(command -v locale-gen)" ]; then
 		locale-gen --destdir "$ROOT"/
 	fi || die "locale-gen failed"
 fi
-
-# Why are we removing these? Don't we need them for final make.conf?
-for useexpand in ${clst_HOSTUSEEXPAND}; do
-	x="clst_${useexpand}"
-	sed -i "/${useexpand}=\"${!x}\"/d" \
-	${clst_make_conf}
-done
-
-# Clear USE
-[ -e ${clst_make_conf} ] && sed -i -e "/^CATALYST_USE/d" ${clst_make_conf}
-[ -e ${clst_make_conf} ] && sed -i -e "/^USE=\"/s/\${CATALYST_USE} ${USE} ${BOOTSTRAP_USE}//" ${clst_make_conf}
diff --git a/targets/stage1/controller.sh b/targets/stage1/controller.sh
index 0db1614d..974ff0fa 100755
--- a/targets/stage1/controller.sh
+++ b/targets/stage1/controller.sh
@@ -11,7 +11,8 @@ case "$1" in
 		install -d "${clst_stage_path}/${clst_port_conf}"
 
 		# Setup make.conf and make.profile link in "ROOT in chroot":
-		copy_to_chroot "${clst_chroot_path}${clst_make_conf}" "${clst_root_path}${clst_port_conf}"
+		[ -e "${clst_chroot_path}${clst_make_conf}" ] && \
+			copy_to_chroot "${clst_chroot_path}${clst_make_conf}" "${clst_root_path}${clst_port_conf}"
 
 		# Enter chroot, execute our build script
 		exec_in_chroot \
diff --git a/targets/support/chroot-functions.sh b/targets/support/chroot-functions.sh
index 08738d0a..f1af451e 100755
--- a/targets/support/chroot-functions.sh
+++ b/targets/support/chroot-functions.sh
@@ -64,19 +64,12 @@ setup_features() {
 	then
 		features+=(distcc)
 		export DISTCC_HOSTS="${clst_distcc_hosts}"
-		[ -e ${clst_make_conf} ] && \
-			echo 'USE="${USE} -avahi -gtk -gnome"' >> ${clst_make_conf}
-		# We install distcc to / on stage1, then use --noreplace, so we need to
-		# have some way to check if we need to reinstall distcc without being
-		# able to rely on USE, so we check for the distcc user and force a
-		# reinstall if it isn't found.
 		if [ "$(getent passwd distcc | cut -d: -f1)" != "distcc" ]
 		then
-			ROOT=/ run_merge --oneshot sys-devel/distcc
+			USE="${USE} -avahi -gtk -gnome" ROOT=/ run_merge --oneshot sys-devel/distcc
 		else
-			ROOT=/ run_merge --oneshot --noreplace sys-devel/distcc
+			USE="${USE} -avahi -gtk -gnome" ROOT=/ run_merge --oneshot --noreplace sys-devel/distcc
 		fi
-		sed -i '/USE="${USE} -avahi -gtk -gnome"/d' ${clst_make_conf}
 		mkdir -p /etc/distcc
 		echo "${clst_distcc_hosts}" > /etc/distcc/hosts
 
@@ -214,10 +207,6 @@ cleanup_stages() {
 			;;
 	esac
 
-	# Remove bindist from use
-	sed -i "/USE=\"\${USE} bindist\"/d" "${clst_make_conf}"
-	sed -i "/USE=\"bindist\"/d" "${clst_make_conf}"
-
 	rm -f /var/log/emerge.log /var/log/portage/elog/*
 }
 
